src/components/Header/MobileHeader.js

```diff
@@ -288,13 +288,15 @@ class MobileHeader extends React.Component {
       buttonLabel = 'Close Search';
     }
 
+    // The desired initialFocus selector only exists when active:
+    const initialFocus = active ? `.${this.props.className}-searchForm-legend` : null;
```

Comment:
Did this throw errors for you? I think I had the same situation (where the initial focus selector only existed when it was active) but it never threw an error.

Reply:
Yes - but not in browser - only in `npm test`. I think it's something peculiar about `enzyme` dom emulation, maybe? I couldn't not sort out why it's happening. It makes little sense because `FocusTrap` should only seek the `initialFocus` element when `active` is `true`. The targeted node is made available when `active` is `true`. So the node should be available in the same cycle that `FocusTrap` becomes `active`, but it somehow seems to be seeking the `initialFocus` node before that part of the dom is built..

Reply:
Got it. 👍